container-images/scripts/build_llama_and_whisper.sh

```diff
@@ -38,12 +38,10 @@ dnf_install() {
 }
 
 cmake_steps() {
-  local cmake_flags=("${!1}")
+  local cmake_flags=("$@")
   cmake -B build "${cmake_flags[@]}"
   cmake --build build --config Release -j"$(nproc)"
-  cmake --install build --prefix=/usr
-  #HACK libggml-cpu.so is being installed in the wrong place should be in /usr/lib64
```

Comment:
ldconfig removes this hack basically

Reply:
Why not install in /usr rather then in /usr/local?

Reply:
It is installing in /usr (rather than /usr/local) via:

"-DCMAKE_INSTALL_PREFIX=$install_prefix"

it was broken before though you were correct.

I built this all locally and checked.

In the Nvidia build $install_prefix is a temporary directory as we have separate build-time and run-time images